### PR TITLE
Adding Podia

### PR DESCRIPTION
**Pull request by chrismitchell**
Hi @alextaylor000 here is a PR adding Podia to the list. 

Thanks

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -265,3 +265,13 @@
     version: 5
     description: >
       SwingTradeBot's core application is powered by Rails and a PostgreSQL database.
+
+- name: Podia
+  url: https://www.podia.com
+  logo_url: https://www.dropbox.com/s/srkqnjlm2xgzopu/podia-wm-purple-medium.png
```
Comment (mctaylorpants) [inline _data/apps.yml]: I think Dropbox is rendering this image within an HTML page, so it's not loading correctly when required as an image. OK if we replace it with the logo from Twitter?

`https://pbs.twimg.com/profile_images/938423381891850240/OSO-VUJY_400x400.jpg`
twitter

**Review by mctaylorpants — CHANGES_REQUESTED** [review]
Thanks for the PR! One small note on the logo.

File: _data/apps.yml (hunk 1 above)
Comment (chrismitchell) [reply _data/apps.yml]: @alextaylor000 yeah that would be perfect thanks for the heads up. Want me to update the URL?

**Review by mctaylorpants — APPROVED** [review]
All good, just wanted to make sure you were ok with the logo change 😄 thanks!

**Pull request merged**